### PR TITLE
fix quick start steps in README

### PR DESCRIPTION
**Pull request by aleksa2808**

### CODE REVIEW

**Review by dyastremsky — DISMISSED** [review]
Thanks! These changes are good. Approved.

File: README.md [hunk 1]
```diff
@@ -71,8 +71,9 @@ dotnet publish -c Release
 ##### You have supported hardware and drivers and [NVIDIA-Docker].
 
 ```
-git clone https://github.com/NVIDIA/model-analyzer
-docker build -f Dockerfile -t model-analyzer .
+git clone https://github.com/triton-inference-server/model_analyzer
+cd model_analyzer
+docker build -t model-analyzer .
```
Comment (whoisj) [inline README.md:76]: should be `docker build -t memory-analyzer`

Comment (aleksa2808) [reply README.md]: changed

**Review by whoisj — APPROVED** [review]

**Comment by whoisj** [comment]
approved and merged 🎉 

thanks @aleksa2808 !

**Pull request merged**